.github/workflows/linux_post_commit.yml

```diff
@@ -27,7 +27,7 @@ jobs:
                  ;;
              SharedLibs)
                  wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
-                 sudo add-apt-repository "deb http://apt.llvm.org/bionic/   llvm-toolchain-bionic main"
+                 sudo add-apt-repository "deb http://apt.llvm.org/focal/   llvm-toolchain-focal main"
```

Comment:
We are running CI on `ubuntu-latest`, which is Ubuntu 18.04 according to GitHub CI documentation: https://github.com/actions/virtual-environments.
What is the problem you are trying to solve?

Reply:
@bader that page seems to be outdated. Job logs say the OS is Ubuntu 20.04: https://pipelines.actions.githubusercontent.com/Ui2fuvl1bHoUfhi59S85MoNTez4CTvidlbaq73AYQSejBPQAOX/_apis/pipelines/1/runs/13643/signedlogcontent/5?urlExpires=2021-03-03T09%3A30%3A09.2445892Z&urlSigningMethod=HMACV1&urlSignature=aRuGTxAuPdF0adpC2XF7fZInwsWmzCe1imhta9FAwhw%3D

Reply:
Probably it would be better to set OS version explicitly instead of using `ubuntu-latest`, since we have hardcoded path to Ubuntu repository.

Reply:
@bader would you prefer using `ubuntu-20.04` or `ubuntu-18.04`?

Reply:
I don't think it really matters for DPC++.
Let's set ubuntu-18.04 as it's tested and switch to ubuntu-20.04 in a separate PR.

Reply:
Ok, updated to explicitly use `ubuntu-18.04`

Reply:
> @bader would you prefer using `ubuntu-20.04` or `ubuntu-18.04`?

Why choosing between these while you can get `ubuntu-20.10`? ;-)

Reply:
Is `ubuntu-20.10` supported by GitHub Actions? It looks like they support only LTS releases.